data-prepper-core/src/main/java/com/amazon/dataprepper/DataPrepper.java

```diff
@@ -60,6 +78,7 @@ public void shutdown() {
         transformationPipelines.forEach((name, pipeline) -> {
             pipeline.shutdown();
         });
+        dataPrepperServer.stop();
```

Comment:
nit: what happens when customer wants to shutdown 1 of the 1 pipeline running using its name - I think the disconnect is in the existing code. [just have this in back of ur mind]

Reply:
Yeah, I think we need to define what the behavior should be. Potentially the server can still run if all the pipelines are shut down.